### DISCUSSION TITLE
AH2 Episode Selector

### DISCUSSION THREAD

**Discussion by TechXXX** [umbrellaplug/umbrellaplug.github.io#185] [General]
Hi guys, when using the episode selector after watching the first episode of a show from an Umbrella widget it uses tmdb helper to start the second episode and I get the 'building player dialogue' and its lag. I thought this was an AH2 issue and so I raised the issue on Jurialmunkeys github in order to improve the situation, he said it's something the native addon should do: 

https://github.com/jurialmunkey/skin.arctic.horizon.2/issues/739

As per usual any help will be mightily appreciated.

**Comment by umbrellaplug** [comment]
You seem to be trying to report issues in the discussions area.

**Comment by TechXXX** [comment]
Seems like I'm not getting away with it

**Reply by umbrellaplug** [reply]
I just thought maybe you were lost. :)

**Reply by TechXXX** [reply]
nah thought I'd give it a shot, I'll repost it as an issue - then again, the issues tab seems to have disappeared from this github EDIT: saw your post on reddit, no worries, no hurries.
